public/js/plugins/bootstrap-input-spinner.js

```
@@ -4,330 +4,325 @@
-      buttonsClass: "btn-outline-secondary",
-      buttonsWidth: "2.5rem",
-      textAlign: "center",
-      autoDelay: 500, // ms holding before auto value change
-      autoInterval: 100, // speed of auto value change
-      boostThreshold: 10, // boost after these steps
-      boostMultiplier: "auto", // you can also set a constant number as multiplier
+    var triggerKeyPressed = false;
+    var originalVal = $.fn.val;
+    $.fn.val = function (value) {
+        if (arguments.length >= 1) {
+            if (this[0] && this[0]['bootstrap-input-spinner'] && this[0].setValue) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)